### PR TITLE
Simplify README, specify review level and platform goals.

### PR DESCRIPTION
**Pull request by gregory-marton**

### CODE REVIEW

**Comment by gregory-marton** [comment]
@raxraxraxraxrax would appreciate a brief review.

File: README.md [hunk 1]
```diff
@@ -49,20 +33,23 @@ $ python setup.py install
...
-If you are using the bayeslite shell, load the bayeslite shell
-commands with:
-
-```
-.hook /path/to/bdbcontrib/hooks/hook_plotting.py
-```
+Our compatibility aim is to work on probcomp machines and members'
+laptops, and to provide scripts and instructions that make it not too
+hard to re-create our environments elsewhere. Polished packaging,
+broad installability without much work, etc. are anti-goals, because
```
Comment (fsaad) [inline README.md]: I find the terms "anti-goal" (and at one point "anti-priority") unnecessary. We should just say "is not a goal" or "not a priority". Moreover, it is not true that proper packaging and instability are incompatible with our core mission of probabilistic computing as the statement implies -- they are just not the highest priority right now.

Comment (gregory-marton) [inline README.md]: @fsaad Respectfully disagree. They are not things we should be paying any attention to at this time. This is not a permanent change, but I want to limit our focus for the time being. Those will become appropriate goals again roughly when moderate review becomes appropriate again.

Comment (fsaad) [inline README.md]: The problem is the awkward use of the word "anti-goal", which [means](https://en.wiktionary.org/wiki/antigoal): _An undesirable state to be avoided_.

Clearly it is __not__ undesirable for the packaging to be highly polished and installable; it undesirable for us to be _spending time_ doing that, which your current README does not make clear. If you want to use the word "anti-goal" we should be very specific about what exactly is the "anti-goal".

**Comment by gregory-marton** [comment]
Attempted to be clearer about that.

File: README.md [hunk 2]
```diff
@@ -1,43 +1,27 @@
...
 
 ```
 $ ./check.sh
 ```
 
+That does not run the complete test suite, only the smoke tests, but
+is usually good enough. For the full suite:
+
+```
+$ ./check.sh tests shell/tests
```
Comment (riastradh-probcomp) [inline README.md]: Omit `shell/tests` -- it does not exist here, and if you include it then no tests run but pytest indicates success!

Comment (gregory-marton) [inline README.md]: Thanks, done.

**Comment by raxraxraxraxrax** [comment]
Looks good to me. Non-actionable commentary: I think it communicates "This is research software, and unless you want to play with it specifically, you should probably go try something else for now," which is our goal at this time. If/when that goal changes, we should be sure to come back and make this more welcoming.

**Pull request merged**